code/src/LikeButton.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import './LikeButton.css';
+
+
+export const LikeButton = props => {
+  const [clicks, setClicks] = useState((props.hearts));
+
+  if(!localStorage[props.id]) {
+    localStorage.setItem(props.id, 0)
+  }
+
+  const handleClick = () => {
+
+    fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts`, {
```

Comment:
This is not proper URL for like button, you never send POST request to API in regards to increase amount of likes by one. Please, take a look on project brief and find proper endpoint (URL) for that

Reply:
Fixed.